src/NHibernate/Async/Id/Insert/IBinder.cs

```diff
@@ -0,0 +1,20 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by AsyncGenerator.
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+
+using System.Data.Common;
+
+namespace NHibernate.Id.Insert
+{
+	using System.Threading.Tasks;
+	public partial interface IBinder
```

Comment:
All generated files that are not included in the project should be deleted. This can occur when excluding methods or types from generation after the generated files were commited. The generator before start analyzing will only exclude all files inside the "Async" folder of a project. I manually deleted all Async folders and regenerated the code to be sure if there are more unused files, but they aren't (only the two mentioned).